src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -179,6 +179,9 @@ def _load_refcase(refcase_file: Optional[str]) -> Optional[EclSum]:
         if refcase_file is None:
             return None
 
+        if refcase_file.endswith(".DATA"):
+            refcase_file = refcase_file[:-5]
```

Comment:
I guess this comment applies here as well: https://github.com/equinor/ert/pull/5247#discussion_r1169775718